docs/source/glossary.rst

```diff
@@ -4,50 +4,61 @@ Glossary
 
   Validator
-    Component of a node responsible for interactions with the blockchain.
-    Interactions with the validator are abstracted by the Consensus API.
+    Sawtooth component that is responsible for interactions with the
+    blockchain. The validator interacts with the consensus engine through the
+    consensus API.
 
   Block
-    A part of the
-    `blockchain <https://en.wikipedia.org/wiki/Block chain>`__,
-    containing some operations and a link to the previous block.
+    Part of the `blockchain <https://en.wikipedia.org/wiki/Block chain>`__
+    that contains one or more operations (called `transactions`) and a link to
+    the previous block.
+
+  Primary node
+    Node that directs the consensus process for the network. (The other nodes
+    in the network participate as :term:`secondary nodes <secondary node>`.)
+    The primary node receives each block from the consensus API, publishes it
+    to all nodes on the network, and starts the consensus process for the block.
+    It also makes the final consensus decisions and commits blocks to the
+    blockchain.
 
-  Primary
-    Node in charge of making the final consensus decisions and committing to
-    the blockchain.  Additionally is responsible for publishing the blocks given
-    to it by the Consensus API, and starting the consensus process.
+    The `original PBFT paper <http://pmg.csail.mit.edu/papers/osdi99.pdf>`__
+    uses the term `leader` instead of `primary`.
+
+  Secondary node
+    Auxiliary node used for consensus. A Sawtooth network using PBFT consensus
+    has one :term:`primary node`; all other nodes are secondary nodes.
 
-  Secondaries
-    Auxiliary nodes used for consensus.
+    The `original PBFT paper <http://pmg.csail.mit.edu/papers/osdi99.pdf>`__
+    uses the terms `follower` or `backup` instead of `secondary`.
 
   Client
     Machine that sends requests to and receives replies from the network of
-    nodes. PBFT has no direct interaction with clients; the Validator bundles
-    all client requests into blocks and sends them through the Consensus API to
-    the consensus algorithm.
+    Sawtooth nodes. PBFT has no direct interaction with clients; the validator
+    bundles all client requests into blocks and sends them through the
+    consensus API to the consensus engine.
 
   Consensus seal
-    Proof that a block underwent consensus.
+    Proof that a block went through consensus.
 
   Member node
     Sawtooth node that participates in PBFT consensus. Membership is controlled
     by the on-chain setting ``sawtooth.consensus.pbft.members``.
 
   Message
-    Block, with additional information (see :ref:`pbft-arch-message-types`).
+    Block with additional consensus-related information (see
+    :ref:`pbft-arch-message-types`).
 
   Working block
-    The block that has been initialized but not finalized, and is currently
-    being committed to.
+    Sawtooth block that has been initialized (but not finalized) and is
```

Comment:
"Working block" is old terminology; PBFT no longer explicitly tracks this, so this term can be removed.

Reply:
Removed this entry.